contracts/CO30_Jonty_JackalsLoot.Woodland_ACR/initPlayerLocal.sqf

```diff
@@ -0,0 +1,48 @@
+#include "do_not_edit\script_component.hpp"
+#include "do_not_edit\initPlayerLocal.sqf"
+
+[
+    // Add additional files here
+    // if you create "edit_me/briefing/example.html, then add "Example" here
+] call mission_fnc_briefing;
+
+[convoyOff, 2, false, 1, "Get Intel", [], 5,
+"Missing Inventory",
+"We got hit bad. Miyović's men ambushed us in this damn forest. We were hoping to avoid attention by taking these secluded roads but it made the perfect opportunity for Miyović's guys to raid our assets. These trucks were transporting vital military assets – missile and air-to-ground bombing units. Miyović, that psycho, might not be able to deploy these complex weapons systems himself, but we fear his forces could extract the deadly warheads and unleash catastrophic damage. We gotta get those missiles secured before they fall into the wrong hands. UAV surveillance has tracked high activty in the red zone marked on the map. That may be a good place to start your search."
+] call zen_modules_fnc_addIntelAction;
```

Comment:
The intel should really just be the most important info, with jot notes (or at least jot note summary after some text).

For interactions with people, we want to use audio over text